From b63466c99a7fd940eb67b1d29d7458f46f6f3e66 Mon Sep 17 00:00:00 2001
From: pdiopX <piotrx.diop@intel.com>
Date: Wed, 9 Jan 2013 11:53:55 +0100
Subject: [PORT FROM MAIN][AUDIO] Fix AudioPolicyManagerBase compilation warnings

This patch fixes -Wsign-compare -Wreorder and -Wenum-compare warnings during
AudioPolicyManagerBase.cpp compilation.

Change-Id: I0c49be0b4cc679bedfbddf2d087abfc11fb943b0
Orig-Change-Id: I474bc574012e2b3c41e7d0b8ebdb0899fbe86261
Signed-off-by: pdiopX <piotrx.diop@intel.com>
---
 audio/AudioPolicyManagerBase.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/audio/AudioPolicyManagerBase.cpp b/audio/AudioPolicyManagerBase.cpp
index 85ddb94..84491f4 100644
--- a/audio/AudioPolicyManagerBase.cpp
+++ b/audio/AudioPolicyManagerBase.cpp
@@ -333,7 +333,7 @@ void AudioPolicyManagerBase::setPhoneState(int state)
         newDevice = hwOutputDesc->device();
     }
 
-    int delayMs = 0;
+    uint32_t delayMs = 0;
     if (isStateInCall(state)) {
         nsecs_t sysTime = systemTime();
         for (size_t i = 0; i < mOutputs.size(); i++) {
-- 
1.7.4.1

